src/screens/FeedScreen/Header.js

```diff
@@ -312,9 +407,12 @@ const Header = ({
     height,
     source,
     headerStyle,
-    onHeaderOptionClicked: () => onHeaderOptionClicked(props),
-    hideThreeDot,
-    isPostDetail
+    isPostDetail,
+    isFollow,
+    onPressFollUnFoll,
+    onDeletePost,
+    isShowDelete,
+    isSelf
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new props are also passed to the `_renderProfileNormal` function, similar to the `_renderProfileAnonymous` function. This change is consistent and necessary for the new features being implemented.
```diff
+    isPostDetail,
+    isFollow,
+    onPressFollUnFoll,
+    onDeletePost,
+    isShowDelete,
+    isSelf
```

447:
The `color` property is added to the `feedUsername` style. This change does not affect the functionality of the code but may improve the visual appearance of the username.
```diff
+    color: colors.black
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->